src/index.ts

```diff
@@ -249,19 +251,87 @@ const generateServerCurseForgeCommand: CommandModule = {
 
         const minecraftVersion = new MinecraftVersion(modpackManifest.minecraft.version)
 
-        // Extract forge version
-        // TODO Support fabric
-        const forgeModLoader = modpackManifest.minecraft.modLoaders.find(({ id }) => id.toLowerCase().startsWith('forge-'))
-        const forgeVersion = forgeModLoader != null ? forgeModLoader.id.substring('forge-'.length) : undefined
-        logger.debug(`Forge version set to ${forgeVersion}`)
+        // Extract forge or fabric version
+        const primaryModLoader = modpackManifest.minecraft.modLoaders.find(({ primary }) => primary)
+        let forgeVersion;
+        let fabricVersion;
+        if (primaryModLoader != null) {
+            if (primaryModLoader.id.includes("forge")) {
```

Comment:
Wouldn't that returns true if modloader is "neoforge"?
![image](https://github.com/dscalzi/Nebula/assets/45696571/9dddb0f7-259b-49d4-94c7-0f2d6822539b)